api/descriptions/signup/signup.md

```diff
@@ -1 +1,5 @@
-Automate Console set up by creating an initial user after Twistlock is first installed.
+Creates the initial admin user after the Console is first installed, to help automation of the Console setup. Invoke this endpoint after Twistlock is first installed.
```

Comment:
@KICKdesigns For example, if I needed to choose between `Prisma Cloud` and `Prisma Cloud Compute`, I'd use `Prisma Cloud Compute` here.